GCSViews/ConfigurationView/ConfigRawParams.cs

```diff
@@ -277,10 +317,11 @@ private void BUT_writePIDS_Click(object sender, EventArgs e)
                     }
 
                     MainV2.comPort.setParam(value, (double)_changes[value]);
+
                     //check if reboot required
-                    if (ParameterMetaDataRepository.GetParameterRebootRequired(value, MainV2.comPort.MAV.cs.firmware.ToString()))
-                    {
-                        reboot = true;
+                    if (ParameterMetaDataRepository.GetParameterRebootRequired(value, MainV2.comPort.MAV.cs.firmware.ToString()))
+                    {
+                        reboot = true;

```

Comment:
Drop the whitespace changes

Reply:
dropped in [commit id 22c75ed](https://github.com/ArduPilot/MissionPlanner/pull/3412/commits/22c75ed012c5702bd9eb70df164d8166320fa3e3)